vsintegration/src/FSharp.Editor/LanguageService/WorkspaceExtensions.fs

```diff
@@ -0,0 +1,209 @@
+﻿[<AutoOpen>]
+module internal Microsoft.VisualStudio.FSharp.Editor.WorkspaceExtensions
+
+open System
+open System.Runtime.CompilerServices
+open System.Threading
+open Microsoft.CodeAnalysis
+open FSharp.Compiler
+open FSharp.Compiler.CodeAnalysis
+
+[<AutoOpen>]
+module private CheckerExtensions =
+
+    type FSharpChecker with
+        member checker.ParseDocument(document: Document, parsingOptions: FSharpParsingOptions, userOpName: string) =
```

Comment:
Please put `///` comments on each member in this file

Reply:
Will do.